sdk/nodejs/runtime/template/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "template",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
```

Comment:
Wrong license, should be Apache 2.0.

Reply:
Fixed in [4bd20c2](https://github.com/dagger/dagger/pull/6220/commits/4bd20c24db7819142b3c865f31fd761648eee4a0)